climpred/bootstrap.py

```diff
@@ -18,6 +23,57 @@
 )
 
 
+def _resample(hind, shuffle_dim, to_be_shuffled):
+    """Resample with replacement in dimension `shuffle_dim` from values of
+    `to_be_shuffled`
+
+    Args:
+        hind (xr.object): input xr.objext to be shuffled.
+        shuffle_dim (str): dimension to shuffle along.
+        to_be_shuffled (list, xr.DataArray.values, np.ndarray): values to shuffle from.
+
+    Returns:
+        xr.object: shuffled along `shuffle_dim`.
+
+    """
+    smp = np.random.choice(to_be_shuffled, len(to_be_shuffled))
+    smp_hind = hind.sel({shuffle_dim: smp})
+    # ignore because then inits should keep their labels
+    if shuffle_dim != 'init':
+        smp_hind[shuffle_dim] = np.arange(1, 1 + smp_hind[shuffle_dim].size)
+    return smp_hind
+
+
+def my_quantile(ds, q=0.95, dim='bootstrap'):
+    """Faster but same results xr.quantile that also allowes lazy `ds`."""
```

Comment:
Can you add to Args, etc. to the docstring? Maybe for conciseness can make the leading description "Computes the quantile ``q``, allowing for lazy computation."